pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vsphere_interface_connexion/guide.fr-fr.md

```diff
@@ -1,42 +1,121 @@
 
-Les identifiants sont envoyés par e-mail durant la création du Hosted Private Cloud, lors d’une modification du mot de passe ou à la création d’un utilisateur :
+> [!primary]
+> Si vous n'êtes pas familier avec l'utilisation de l'API OVHcloud, consultez notre guide « [Premiers pas avec les API OVHcloud](/pages/manage_and_operate/api/first-steps)».
 
-```
-adresse IP/Nom : pcc-xxx-xxx-xxx-xxx.ovh.com nom d’utilisateur : admin mot de passe : xxxxxx
-```
+Voici quelques exemples d'appels API :
 
-Ce document de VMware répertorie les différents ports que vous devez ouvrir sur votre pare-feu pour, par exemple, accéder à la console : [Accès console](https://kb.vmware.com/kb/1012382){.external-link}
+- Créer un utilisateur :
 
-### Utilisation du web client HTML5
+> [!api]
+>
+> @api {v1} /dedicatedCloud POST /dedicatedCloud/{serviceName}/user
+>
+
+> **Paramètres** :
+> 
+> - `serviceName` : votre service sous la forme pcc-XX-XX-XX-XX.
+> - `name` : le nom de votre utilisateur sous la forme ci-dessous.
+> 
+> Exemple :
+>
+> ```shell
+> {
+> "name": "User name"
+> }
+> ```
+
+- Changer le mot de passe de votre utilisateur :
+
+> [!api]
+>
+> @api {v1} /dedicatedCloud POST /dedicatedCloud/{serviceName}/user/{userId}/changePassword
+>
+
+> **Paramètres** :
+>
+> - `serviceName` : votre service sous la forme pcc-XX-XX-XX-XX.
+> - `password` : le nom de votre utilisateur sous la forme ci-dessous.
+>
+> Exemple :
+>
+> ```shell
+> {
+> "password": "XXX"
+> }
+> ```
+
+### Étape 2 - Connexion à l'interface web vSphere
+
+**Utilisation du web client HTML5**
 
 Le web client HTML5 est disponible sur l’interface web de votre Hosted Private Cloud à l’adresse : <https://pcc-xxx-xxx-xxx-xxx.ovh.com/ui> (remplacez pcc-xxx-xx-xx-xxx.ovh.com par l’adresse de votre Hosted Private Cloud).
 
-![Connexion à l'interface vSphere HTML5](images/connection_interface_w_html5.png){.thumbnail}
+![Connexion à l'interface vSphere HTML5](images/vsphere_web_client_all.png){.thumbnail}
 
 Vous accéderez ensuite à cette interface :
 
-![Connexion à l'interface vSphere HTML5](images/vsphere-client-html5.png){.thumbnail}
+![Connexion à l'interface vSphere HTML5](images/vsphere_web_client_iam_vs_local.png){.thumbnail}
+
+Via un utilisateur IAM :
+
+![Connexion à l'interface vSphere HTML5](images/vsphere_web_client_iam.png){.thumbnail}
+
+Via un utilisateur local :
+
+![Connexion à l'interface vSphere HTML5](images/vsphere_web_client_local.png){.thumbnail}
 
 La page `Home`{.action} permet de retrouver les grands menus de votre vCenter.
 
+![Connexion à l'interface vSphere HTML5](images/vsphere_web_client_pcc_home.png){.thumbnail}
+
 ## Aller plus loin
 
-Échangez avec notre communauté d'utilisateurs sur <https://community.ovh.com>.
+Ce document de VMware répertorie les différents ports que vous devez ouvrir sur votre pare-feu pour, par exemple, accéder à la console : [Accès console](https://kb.vmware.com/kb/1012382).
```

Comment:
@pg-ovh , I've moved this line at the bottom of the guide as it became unrelated to the part it was in. Feel free to remove it if it's not needed anymore.